lib/matplotlib/tests/test_backends_interactive.py

```diff
@@ -0,0 +1,49 @@
+import os
+import sys
+
+from matplotlib.compat.subprocess import Popen
+import pytest
+
+
+# Minimal smoke-testing of the backends for which the dependencies are
+# PyPI-installable on Travis.  They are not available for all tested Python
+# versions so we don't fail on missing backends.
+#
+# Once the Travis build environment switches to Ubuntu 14.04, we should be able
+# to add wxagg (which has wheels for 14.04 but not for 12.04).
+
+
+def _get_available_backends():
+    backends = []
+    for module_name, backend in [
+            ("PyQt5", "qt5agg"),
+            ("Tkinter", "tkagg"),
+            ("tkinter", "tkagg")]:
+        try:
+            __import__(module_name)
```

Comment:
I would have used importlib.import_module if we were Py3-only, but to be honest I had lost track of what Py2 supports.  Anyways, I'll fix it.
I think GUI frameworks "should" only install an inputhook when their mainloop is started, which is not the case here.  To be honest I'd rather not have to add version-specific code unless strictly necessary.